docs/recipes.rst

```diff
@@ -695,45 +510,13 @@ Data formats which can be added are :-
                         "name": "zip",
                         "type": "int"
                     },
-                    {
-                        "name": "colorectal",
-                        "type": "double"
-                    },
-                    {
-                        "name": "lung_bronc",
-                        "type": "double"
-                    },
-                    {
-                        "name": "breast_can",
-                        "type": "double"
-                    },
-                    {
-                        "name": "prostate_c",
-                        "type": "double"
-                    },
-                    {
-                        "name": "urinary_sy",
-                        "type": "double"
-                    },
-                    {
-                        "name": "all_cancer",
-                        "type": "double"
-                    },
-                    {
-                        "name": "shape_length",
-                        "size": "50,30",
-                        "type": "decimal"
-                    },
-                    {
-                        "name": "shape_area",
-                        "size": "50,30",
-                        "type": "decimal"
-                    }
-                ]
-            },
+
     ...
 
-5. HDF5 Data :- To make a script for HDF5 data add object ``hdf5_data`` object as shown in the script ``sample-hdf`` for `Sample HDF5` dataset. Add the name of the xml file, data type and group of the data in the ``xml_data`` object as a list.
+5. HDF5 Data :- To make a script for HDF5 data add object ``hdf5_data``
```

Comment:
For HDF5 raw data, add the  key word `hdf5_data` to the resource. The value to the key is the .....